RatRazor/Pages/Overlay/Index.razor

```diff
@@ -112,6 +121,26 @@
         return url;
     }
 
+    // We need to offset the tooltip position by the absolute position of the game (thus the overlay)
+    private Point GetItemOverlayPosition(ItemScan itemScan)
```

Comment:
This is where the relative position with the overlay is calculated. In theory this could be optimized by moving this inside the scan result generation.